templates/21-loggly-base.conf

```diff
@@ -8,7 +8,7 @@ $PrivDropToGroup adm
 $WorkDirectory /var/spool/rsyslog # where to place spool files
 
 $template LogglyFormatNginx,"<%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [{{ loggly.token }}@41058 tag=\"nginx\"] %$!msg%\n"
-$template LogglyFormat{{ loggly.application.tag }},"<%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [{{ loggly.token }}@41058 tag=\"{{ loggly.application.tag }}\"] %$!msg%\n"
+$template LogglyFormat{{ hostvars[inventory_hostname]['ec2_tag_app_name'] }},"<%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [{{ loggly.token }}@41058 tag=\"{{ hostvars[inventory_hostname]['ec2_tag_app_name'] }}\"] %$!msg%\n"
```

Comment:
ok, changed the tag in vars in ansible-vpc, so I could go back to loggly.application.tag here